protocol/x/accountplus/authenticator/signature_authenticator.go

```diff
@@ -56,6 +56,11 @@ func (sva SignatureVerification) Initialize(config []byte) (types.Authenticator,
 // Authenticate takes a SignaturesVerificationData struct and validates
 // each signer and signature using signature verification
 func (sva SignatureVerification) Authenticate(ctx sdk.Context, request types.AuthenticationRequest) error {
+	// First consume gas for verifying the signature
+	params := sva.ak.GetParams(ctx)
+	ctx.GasMeter().ConsumeGas(params.SigVerifyCostSecp256k1, "secp256k1 signature verification")
```

Comment:
added gas here @teddyding .

we currently only support secp256k1 signature verification so charging some static gas here.

Reply:
Could we documment the fact that we only supports `secp256k1`? For example link to [here](https://github.com/dydxprotocol/v4-chain/blob/301b4385069765c443f1dc3816dd1229bf24ab0f/protocol/x/accountplus/authenticator/signature_authenticator.go#L93)